README.md

```diff
@@ -1,9 +1,31 @@
-# sgeop
+# `sgeop`: Street Geometry Processing Toolkit
+
+[![Continuous Integration](https://github.com/uscuni/sgeop/actions/workflows/testing.yml/badge.svg)](https://github.com/uscuni/sgeop/actions/workflows/testing.yml)
 
 Street geometry processing toolkit
 
 ```py
 import sgeop
 
 simplified = sgeop.simplify_network(gdf)
-```
\ No newline at end of file
+```
+
+This package developed & and maintained by:
+* [Martin Fleischmann](https://github.com/martinfleis)
+* [Anastassia Vybornova](https://github.com/anastassiavybornova)
+* [James D. Gaboardi](https://github.com/jGaboardi)
+
+
+## Introduction
+
+## Documentation
+
+## Examples
+
+## Contribution
+
+## Recommended Citation
+
+---------------------------------------
+
+Copyright (c) 2023-, SGeoP Developers
```

Comment:
Is SGeoP capitalisation we want? I'd maybe stick to sgeop but fine with either. cc @anastassiavybornova

Reply:
lowercase please!! ❤️